src/clients/online_update.rb

```diff
@@ -182,8 +182,6 @@ def OnlineUpdateSequence
 
       Progress.NextStage
 
-      OnlineUpdateCallbacks.RefreshAllSources
-
```

Comment:
That leaves us with two `Progress.NextStage` in a row. I wonder what the whole progress report looks like...

```rb
     stages = [
        # progress stage label
        _("Initialize the target system"),
        # progress stage label
        _("Refresh software repositories"),
        # progress stage label
        _("Check for available updates")
      ]
```

So at a point where we "Refresh software repositories" there will be nothing. So they will be refreshed automatically later? Let me test it ...

(10 minutes later)

No. Apparently we have removed the only repo refresh. Now I get a message when the sanity check finds something fishy:
![norepos](https://user-images.githubusercontent.com/102056/199203503-292d68e2-6dd7-427e-8f33-8377e1dd025f.png)

I should add that there **is** an update repo already, shown by `zypper lr`.
If I say Yes, I see the repos.
But if I say No and then go to menu > Configuration > Repositories (Ctrl-R), there are none.

So much for a "simple bugfix", @kobliha :smiling_face_with_tear:

Reply:
ok. so more archeology needed. I'll put it somewhere closer to common workflow.

Reply:
Re-adding `Pkg.SourceStartManager` fixes it